hybrid/pi-monte-carlo/src/lib.rs

```diff
@@ -0,0 +1,22 @@
+extern crate rand;
+
+use rand::Rng;
+
+#[no_mangle]
+pub fn montecarlo_pi(trials: u32) -> f64 {
+    let mut count: u32 = 0;
+    let mut rng = rand::thread_rng();
+
+    for _i in 1..trials {
+        let x: f64 = rng.gen::<f64>();
+        let y: f64 = rng.gen::<f64>();
+        let p: f64 = x * x + y * y;
```

Comment:
I personally don't like such cryptic names. :-) 

What is a "p"? (Looks like it refers to the distance to the center of the dart board.)

Reply:
Done